app/core/Tracker.coffee

```diff
@@ -236,18 +236,18 @@ module.exports = class Tracker extends CocoClass
     return if me.isAdmin() or @shouldBlockAllTracking()
     return unless me.isTeacher(true)
     loadSegmentIo()
-    .then =>
-      @segmentLoaded = true
+    .then (segmentIo) =>
+      @segmentLoaded = segmentIo.length is not 0
       @identify()
-      @trigger 'segment-loaded'
+      @trigger 'segment-loaded' if @segmentLoaded
```

Comment:
It looks like no one listens to this event. Should this trigger be deleted?

Reply:
Agreed, yea should be ok to delete.

Previous commit: https://github.com/codecombat/codecombat/commit/3cc8275fc4a27b368da2e1cc17cc69da7f77ae1a